src/app/scripts/cac/search/cac-typeahead.js

```diff
@@ -128,7 +137,7 @@ CAC.Search.Typeahead = (function ($) {
                 '-75.126531',
                 '39.967842'
             ].join(','),
-            category: 'Address,POI',
+            category: searchCategories,
```

Comment:
So they are.  I'll move the categories to a shared module; the bounds should be shared, too.